90 enable posts and publish an article

nellihol opened the pull request.

atruskie commented on content/posts/2024-recap.md:
```diff
@@ -0,0 +1,32 @@
+- General Update
+---
+
+I hope your 2025 is off to a great start. We at Open Ecoacoustics are excited to keep the community more updated on our activities and progress. This year, we're committed to providing more regular updates and sharing our work through these posts and on social media. 
+
+## Follow Us on BlueSky and LinkedIn 
+
+We are excited to announce the launch of our accounts on [BlueSky](https://bsky.app/profile/openecoacoustics.bsky.social) and [LinkedIn](https://www.linkedin.com/company/open-ecoacoustics/). Follow us to stay in the loop! 
```
this sounds like it should be in the 2025 post

atruskie commented on content/posts/2024-recap.md:
```diff
@@ -0,0 +1,32 @@
+
+### In 2024, we achieved significant milestones: 
+
+-  1.8 million recordings uploaded 
```
```suggestion
- 1.8 million recordings uploaded 
```

atruskie commented on content/posts/2024-recap.md:
```diff
@@ -0,0 +1,32 @@
+### In 2024, we achieved significant milestones: 
+
+-  1.8 million recordings uploaded 
+- 132 years of data duration 
```
```suggestion
- 132 years of data added
```

atruskie commented on content/posts/2024-recap.md:
```diff
@@ -0,0 +1,32 @@
+-  1.8 million recordings uploaded 
+- 132 years of data duration 
+- 186 TB of data uploaded
+-  6.7 million recordings downloaded 
```
```suggestion
- 6.7 million recordings downloaded 
```

atruskie commented on content/posts/2024-recap.md:
```diff
@@ -0,0 +1,32 @@
+- 132 years of data duration 
+- 186 TB of data uploaded
+-  6.7 million recordings downloaded 
+- 196 users 
```
```suggestion
- 196 new users 
```

atruskie commented on content/posts/2024-recap.md:
```diff
@@ -0,0 +1,32 @@
+-  6.7 million recordings downloaded 
+- 196 users 
+
+**This means that 24% of all data uploaded to A2O and Ecosounds was ingested last year!**
```
```suggestion
**This means that 24% of all of our data was uploaded to the A2O and Ecosounds just last year!**
```

atruskie commented on content/posts/2024-recap.md:
```diff
@@ -0,0 +1,32 @@
+
+**This means that 24% of all data uploaded to A2O and Ecosounds was ingested last year!**
+
+Thank you for all our partners, users and contributers being part of this journey so far.
```
```suggestion
Thank you to all our partners, users, and contributors being part of this journey so far.
```

atruskie reviewed (changes requested):
i'd suggest we use some shorter tags, like `News` (instead of `General Update`) and `Software` (instead of `Technical Update`)

The continue reading button looks out of place:

![image](https://github.com/user-attachments/assets/28ec6ee9-d4fb-4e98-b27f-d67aeb1f680e)

Can we do something more subtle, like a wipe: gradient, transparent-to-white, top-to-bottom?


Check figure and image alignment:

![image](https://github.com/user-attachments/assets/2a036e62-7410-4bb0-a2bf-360937b60fc1)

I think the posts page would look better with a font size bump.

nellihol commented:
![image](https://github.com/user-attachments/assets/d429ee0b-461e-4bf6-b9b8-1d13bc99b87c)
![image](https://github.com/user-attachments/assets/4ba153ab-6d43-44c4-869e-9d09c62c87f0)
- Changed button style to something less jarring
- Changed figure caption style
- Increased blog post font styles
- Added hero-image support and styles specific to blog posts
- Improved tile interaction (click on anywhere on the tile to go read the blog post, hover effect)
-

atruskie commented on content/posts/2024-recap.md:
```diff
@@ -0,0 +1,31 @@
+
+Our team continued to enhance platform operations, refine the Open Ecoacoustics data pipeline, and incorporate more analyses into analysis containers.  
+
+## In 2024, we achieved significant milestones: 
```
```suggestion
## 2024 milestones

 In 2024, we achieved significant growth: 
```

atruskie commented on content/posts/2024-recap.md:
```diff
@@ -0,0 +1,31 @@
+- 6.7 million recordings downloaded 
+- 196 new users 
+
+## This means that 24% of all of our data was uploaded to the A2O and Ecosounds just last year!
```
```suggestion
This means that **24% of all of our data was uploaded to the A2O and Ecosounds just last year**!
```
